.github/workflows/runAqa.yml

```diff
@@ -97,9 +101,19 @@ jobs:
       matrix: ${{ fromJson(needs.parseComment.outputs.build_parameters) }}
     steps:
     - uses: AdoptOpenJDK/install-jdk@v1
+      if: matrix.sdk_resource != 'customized'
       with:
         version: ${{ matrix.jdk_version }}
         source: ${{ matrix.sdk_resource }}
+        sourceType: 'buildType'
```

Comment:
sourceType can be automatically infer. Though it doesn't hurt to explicitly set it.